chain/network/src/network_protocol/borsh_conv.rs

```diff
@@ -133,7 +133,9 @@ impl TryFrom<&net::PeerMessage> for mem::PeerMessage {
                 created_at: None,
                 num_hops: Some(0),
             })),
-            net::PeerMessage::Disconnect(r) => mem::PeerMessage::Disconnect(r),
+            net::PeerMessage::Disconnect => {
+                mem::PeerMessage::Disconnect(mem::Disconnect { allow_reconnect: true })
```

Comment:
it doesn't really matter what is the default here (as borsh is not used in prod any more), but please document why you chose "true" just for the sake of completeness.

Reply:
I'm renaming this field to `remove_from_recent_outbound_connections` which should make it more clear how the value is used and why the default should be false.